conf/messages

```diff
@@ -1212,4 +1230,14 @@ incomeSources.cease.UKPropertyEndDate.error.beforeStartDate      = The cessation
 
 incomeSources.ceaseForeignProperty.heading                       = Cease foreign property
 incomeSources.ceaseForeignProperty.checkboxLabel                 = I declare that I no longer get an income from any foreign property.
-incomeSources.ceaseForeignProperty.checkboxError                 = You must declare that you no longer get an income from any foreign property
\ No newline at end of file
+incomeSources.ceaseForeignProperty.checkboxError                 = You must declare that you no longer get an income from any foreign property
+
+#Generic date form messages
+dateForm.hint                                                       = For example, 27 3 2020
```

Comment:
I don't think this is the only one form have date input, so you need to have more details in the prefix i.e
> addbusinessstartdateform.hint ?